modules/jooby-gson/src/main/java/org/jooby/json/GsonRenderer.java

```diff
@@ -223,10 +224,16 @@ public GsonRenderer(final MediaType type, final Gson gson) {
   }
 
   @Override
-  public void render(final Object object, final Context ctx) throws Exception {
+  public void render(final Object value, final Context ctx) throws Exception {
     if (ctx.accepts(this.type)) {
-      ctx.type(this.type)
-          .send(gson.toJson(object));
+        JsonElement jsonElement = gson.toJsonTree(value, value.getClass());
+        if (jsonElement.isJsonPrimitive()) {
```

Comment:
let's not change the way it works today, have a look at Jackson module on how I did it (use a specific renderer when raw is required)

Reply:
was what I'm trying to do, however regardless if I added raw () or not it would get the two.

so this way gets the 2 in a single call new Gzon ()

but I'll see if I can change that.

Reply:
yea, think is better to keep backward compatibility here. Also, there is no need to call `toJsonTree` we can just tests if value is a number/string/boolean/character. This also will add a small overhead compared to `toJsonTree`

Reply:
> we can just tests if value is a number/string/boolean/character. This also will add a small overhead compared to toJsonTree


I did not understand this part of toJsonTree's overhead.

Reply:
simple `instanceof` I'm sure it is faster than calling `.toJsonTree`